component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/internal/TOTPAuthenticatorServiceComponent.java

```diff
@@ -193,4 +194,34 @@ protected void unsetIdentityProviderManagementService(IdpManager idpManager) {
 
 		TOTPDataHolder.getInstance().setIdpManager(null);
 	}
+
+	/**
+	 * Set claim metadata management service implementation.
+	 *
+	 * @param claimManagementService ClaimManagementService
+	 */
+	@Reference(
+			name = "claimManagementService",
+			service = org.wso2.carbon.identity.claim.metadata.mgt.ClaimMetadataManagementService.class,
+			cardinality = ReferenceCardinality.MANDATORY,
+			policy = ReferencePolicy.DYNAMIC,
+			unbind = "unsetClaimMetadataManagementService")
```

Comment:
seems tabs are used instead of 4 spaces for intending

Reply:
this change will be reverted after the migration is completed